lib/gollum/app.rb

```diff
@@ -639,6 +636,13 @@ def show_file(file)
       end
     end
 
+    def load_template
+      template_page = wiki_page('/_Template')
+      template_page = (template_page.page != nil) ? template_page.page.raw_data : 'Template page option is set, but no /_Template page is present or committed.'
+      template_page = Gollum::TemplateFilter.apply_filters(template_page) unless Gollum::TemplateFilter.filters.empty?
```

Comment:
we could leave out the `unless` call here, but it's also fine like this.

Reply:
Yes, you're right. In that case I can also leave out the `getter` method `:filters`, cleaning it up even further. 👍